### DISCUSSION TITLE
What are you using Fields2Cover for?

### DISCUSSION THREAD

**Discussion by Gonzalo-Mier** [Fields2Cover/Fields2Cover#31] [General]
Which are the uses you are making of Fields2Cover? I would like to know :heart_eyes: 
Provide images, videos or anything you want.

**Comment by Marnonel6** [comment]
Hi,

I'm going to use if for drone path planning to survey crop fields! Thank you for this package!

**Reply by Gonzalo-Mier** [reply]
Cool! Have fun with F2C :)

**Comment by samuk** [comment]
Hey I'm using fields2cover in this very WIP project: https://sowbot.co.uk

[Code](https://github.com/Agroecology-Lab/feldfreund_devkit_ros#quick-start)

It generates fields2cover rows from a user-selected map in a WebUI, then converts those to topological nav nodes to be consumed by [Topological Navigation](https://github.com/LCAS/topological_navigation/tree/aoc_refactor)

A short video walkthrough is here https://github.com/Agroecology-Lab/Sowbot_Data/raw/refs/heads/dev/webui/video/Screencast%20From%202026-06-14%2013-03-21.mp4

**Reply by Gonzalo-Mier** [reply]
That is super cool!!!  :open_mouth:  Thanks for sharing :heart:

**Reply by samuk** [reply]
@Gonzalo-Mier be interested to know what you think about contour-aware 3d mapping https://github.com/Agroecology-Lab/feldfreund_devkit_ros/issues/110
